manifests/init.pp

```diff
@@ -113,42 +113,42 @@
-  Optional[Variant[Boolean,Enum['resolve']]]             $multicast_dns,
-  Optional[Variant[Boolean,Enum['allow-downgrade']]]     $dnssec,
-  Optional[Variant[Boolean,Enum['opportunistic', 'no']]] $dnsovertls,
-  Optional[Variant[Boolean,Enum['no-negative']]]         $cache,
-  Optional[Variant[Boolean,Enum['udp','tcp']]]           $dns_stub_listener,
-  Boolean                                                $use_stub_resolver,
-  Boolean                                                $manage_networkd,
-  Enum['stopped','running']                              $networkd_ensure,
-  Boolean                                                $manage_timesyncd,
-  Enum['stopped','running']                              $timesyncd_ensure,
-  Optional[Variant[Array,String]]                        $ntp_server,
-  Optional[Variant[Array,String]]                        $fallback_ntp_server,
-  Boolean                                                $manage_accounting,
-  Hash[String,String]                                    $accounting,
-  Boolean                                                $purge_dropin_dirs,
-  Boolean                                                $manage_journald,
-  Systemd::JournaldSettings                              $journald_settings,
-  Boolean                                                $manage_udevd,
-  Optional[Variant[Integer,String]]                      $udev_log,
-  Optional[Integer]                                      $udev_children_max,
-  Optional[Integer]                                      $udev_exec_delay,
-  Optional[Integer]                                      $udev_event_timeout,
-  Optional[Enum['early', 'late', 'never']]               $udev_resolve_names,
-  Optional[Variant[Integer,String]]                      $udev_timeout_signal,
-  Boolean                                                $manage_logind,
-  Systemd::LogindSettings                                $logind_settings,
-  Hash                                                   $loginctl_users = {},
-  Hash                                                   $dropin_files = {},
-  Hash                                                   $udev_rules = {},
+  Hash[String,Hash[String, Any]]                               $service_limits,
+  Boolean                                                      $manage_resolved,
+  Enum['stopped','running']                                    $resolved_ensure,
+  Optional[Variant[Array[String],String]]                      $dns,
+  Optional[Variant[Array[String],String]]                      $fallback_dns,
+  Optional[Variant[Array[String],String]]                      $domains,
+  Optional[Variant[Boolean,Enum['resolve']]]                   $llmnr,
+  Optional[Variant[Boolean,Enum['resolve']]]                   $multicast_dns,
+  Optional[Variant[Boolean,Enum['allow-downgrade']]]           $dnssec,
+  Optional[Variant[Boolean,Enum['yes','opportunistic', 'no']]] $dnsovertls,
```

Comment:
hey, could you please put `Variant[Boolean,Enum['yes','opportunistic', 'no']]` into it's own custom type? I think that would make the whole thing more readable and we don't need to add so much whitespace everywhere

Reply:
Sounds like a good idea. Just the one custom type for resolved::dnsovertls or should i create more while I'm at it?

Reply:
The custom types are not worth it unless it can be used multiple times. If it is only used once or twice, adding custom types just adds complexity to the module making it harder to understand and maintain.

Reply:
I've reverted all the whitespace changes. The alignment does look a bit weird since they aren't aligned to anything. Saying that the alignment for init.pp and resolved.pp is inconsistent:

init.pp:
` Hash                                                   $dropin_files = {},
  Hash                                                   $udev_rules = {},`

resolved.pp
` Enum['stopped','running'] $ensure                                  = $systemd::resolved_ensure,
  Optional[Variant[Array[String],String]] $dns                       = $systemd::dns,`

The first the variables are aligned, the second the equals are.